source/includes/connect.cpp

```diff
@@ -10,4 +10,51 @@ mongocxx::instance instance{};
 
 mongocxx::uri uri("mongodb://localhost:27017");
 mongocxx::client client(uri);
-// end-client
\ No newline at end of file
+// end-client
+
+// start-atlas
+#include <mongocxx/client.hpp>
+#include <mongocxx/uri.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/exception/exception.hpp>
+
+// Replace the placeholder with your Atlas connection string
+mongocxx::uri uri("<connection string>");
+
+// Create a mongocxx::client with a mongocxx::options::client object to set the Stable API version
+mongocxx::options::client client_options;
+mongocxx::options::server_api server_api_options(k_version_1);
+client_options.server_api_opts(server_api_options);
+mongocxx::client client(uri, client_options);
+
+try
+{
+    // Ping the server to verify that the connection works
+    auto admin = client["admin"];
+    bsoncxx::document::value command = bsoncxx::from_json(R"({"ping": 1})");
```

Comment:
Suggest using `auto` to shorten. Suggest using `make_document` for consistency with other doc examples and to avoid unnecessary cost of parsing JSON strings:

```suggestion
    auto command = make_document(kvp("ping", 1));
```

Suggested use of `make_document` in [C/C++ Driver Code Example Patterns and Style Reference](https://docs.google.com/document/d/1wBsUnT69a7XNTUOZd1agQae_pL1fREl-mcg34QgYaCs/edit?disco=AAABQr3bvIQ)